Dashboard/ViewModels/ConnectionWindowViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Media;
+using ReactiveUI;
+using System;
+using System.ComponentModel;
+
+namespace Dashboard.ViewModels 
+{
+    internal class ConnectionWindowViewModel : ReactiveObject
+    {
+
+        private string _selectedConnectionType;
+
+        public string SelectedConnectionType
```

Comment:
This probably shouldn't be a string, we can only select 3 options. why not use an enum?

Also, you can replace almost this entire class with just
`[ObservableProperty] public string _selectedConnectionType = string.Empty;`